draft-ietf-rats-msg-wrap.md

```diff
@@ -162,7 +162,7 @@ This document reuses the terms defined in {{Section 2 of -senml-cf}}
 
 # Conceptual Message Wrapper Encodings
 
-Two types of RATS Conceptual Message Wrapper (CMW) are specified in this
+Two basic types of RATS Conceptual Message Wrapper (CMW) are specified in this
```

Comment:
What does “basic” add? Is there an “advanced”? What would the difference be?

Reply:
I think of basic in terms of growing complexity.

The resulting flow is:

> _Two _basic_ types are specified ... a further CMW "collection" type that holds together multiple CMW items ... and a CMW "tunnel" type is also defined [...] to allow transporting CBOR CMWs in JSON collections and vice-versa._

Reply:
Reading Ned's comment I finally understand what this means. I would say that since CMW is a tree structure, the leaf types are what you're describing as "basic types". When I see "basic" in text, I just feel like the author is taunting me with what they find easy to understand in a sea of complexity.

Reply:
Please suggest wording that works for someone coming here with a fresh pair of eyes.
I have read it so many times that I can no longer be that person, unfortunately.